contact.ipynb

```diff
@@ -0,0 +1,145 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "780d6f7b",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import csv\n"
```

Comment:
write a import csv file write in aother way ?

Reply:
from csv import reader